src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/GregtechMetaTileEntity_SteamCentrifuge.java

```diff
@@ -0,0 +1,225 @@
+                            { " A~A ", "AEBEA", "ABDBA", "AEBEA", " AAA " },
+                            { " AAA ", "AAAAA", "AAAAA", "AAAAA", " AAA " }, }))
+                .addElement('E', ofFrame(Materials.Steel))
+                .addElement('C', ofBlock(getCasingBlock2(), getCasingMeta2()))
+                .addElement('D', ofBlock(getCasingBlock3(), getCasingMeta3()))
+                .addElement('B', ofBlock(getCasingBlock4(), getCasingMeta4()))
+                .addElement(
+                    'A',
+                    ofChain(
+                        buildSteamInput(GregtechMetaTileEntity_SteamCentrifuge.class).casingIndex(CASING_TEXTURE_ID)
+                            .dot(1)
+                            .build(),
+                        buildHatchAdder(GregtechMetaTileEntity_SteamCentrifuge.class)
+                            .atLeast(SteamHatchElement.InputBus_Steam, SteamHatchElement.OutputBus_Steam)
+                            .casingIndex(CASING_TEXTURE_ID)
+                            .dot(1)
+                            .build(),
+                        onElementPass(x -> ++x.mCasing, ofBlock(getCasingBlock5(), getCasingMeta5()))))
+                .build();
+        }
+        return STRUCTURE_DEFINITION;
+    }
+
+    public Block getCasingBlock2() {
+        return sBlockCasings2;
+    }
+
+    public byte getCasingMeta2() {
+        return 13;
+    }
+
+    public Block getCasingBlock3() {
+        return sBlockCasings3;
+    }
+
+    public byte getCasingMeta3() {
+        return 14;
+    }
+
+    public Block getCasingBlock4() {
+        return sBlockCasings2;
+    }
+
+    public byte getCasingMeta4() {
+        return 3;
+    }
+
+    public Block getCasingBlock5() {
+        return sBlockCasings2;
+    }
+
+    public byte getCasingMeta5() {
+        return 0;
+    }
+
+    public boolean checkMachine(IGregTechTileEntity aBaseMetaTileEntity, ItemStack aStack) {
+        mCasing = 0;
+        fixAllMaintenanceIssue();
+        boolean didBuild = checkPiece(mName, 2, 3, 0);
+        return didBuild && mCasing >= 50 && checkHatch();
+    }
+
+    @Override
+    public void construct(ItemStack stackSize, boolean hintsOnly) {
+        buildPiece(mName, stackSize, hintsOnly, 2, 3, 0);
+    }
+
+    @Override
+    public int survivalConstruct(ItemStack stackSize, int elementBudget, ISurvivalBuildEnvironment env) {
+        if (mMachine) return -1;
+        return survivialBuildPiece(mName, stackSize, 2, 3, 0, elementBudget, env, false, true);
+    }
+
+    @Override
+    protected GT_Multiblock_Tooltip_Builder createTooltip() {
+        GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
+        tt.addMachineType(getMachineType())
+            .addInfo("Controller Block for the Steam Centrifuge")
+            .addInfo("Runs recipes up to MV tier")
+            .addInfo("Centrifuged up to " + getMaxParallelRecipes() + " things at a time")
```

Comment:
```suggestion
            .addInfo("Centrifuges up to " + getMaxParallelRecipes() + " things at a time")
```
Typo. Alternatively use standard wording like `Processes N recipes in parallel`.